packages/rolldown/package.json

```diff
@@ -71,8 +71,7 @@
     "build-browser:debug": "cross-env BROWSER_PKG=1 pnpm run --sequential \"/^build-(binding|binding:wasi|node)$/\"",
     "build-browser:release": "cross-env BROWSER_PKG=1 pnpm run --sequential \"/^build-(binding|binding:wasi:release|node)$/\"",
     "# Scrips for docs #": "_",
-    "extract-options-doc": "typedoc",
-    "preinstall": "npx only-allow pnpm"
```

Comment:
Should made it as a part of `just setup`.

Reply:
Sorry, I don't know how to do this. Could you please continue?